src/de/jost_net/JVerein/keys/Staat.java

```diff
@@ -0,0 +1,120 @@
+/**********************************************************************
+ * Copyright (c) by Heiner Jostkleigrewe
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,  but WITHOUT ANY WARRANTY; without
+ *  even the implied warranty of  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+ *  the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.  If not,
+ * see <http://www.gnu.org/licenses/>.
+ *
+ * heiner@jverein.de
+ * www.jverein.de
+ **********************************************************************/
+package de.jost_net.JVerein.keys;
+
+public enum Staat
+{
+  AD ("ANDORRA"),
```

Comment:
Die enums sind hier nach den Abkürzungen sortiert. Am GUI bekommt man aber den Text zur Auswahl in der Drop Down Liste. Die ist jetzt nicht alphabetisch sortiert.
Kann man die Einträge für die Drop Down Liste alphabetisch sortieren z.B. das Staat.values() überschreiben?

Reply:
habe ich gemacht